Classes/Bookmark/BookmarksViewController.swift

```diff
@@ -37,7 +45,7 @@ TabNavRootViewControllerType {
 
         configureSearchBar()
         configureTableView()
-
+        
```

Comment:
Please change your whitespace settings 😆 

<img width="795" alt="screen shot 2017-10-24 at 9 38 56 pm" src="https://user-images.githubusercontent.com/739696/31976157-c4b8c71a-b903-11e7-85ff-37e30ee1784b.png">

Reply:
Including whitespace-only lines for turned off for me. Thx for the info 👍